src/utils/avatar.js

```diff
@@ -276,36 +276,10 @@ export const getAvatarUrl = (
   email: string,
   realm: URL,
   sizePhysicalPx: number,
-): string => {
-  if (typeof avatarUrl !== 'string') {
-    // It's our job to compute the Gravatar hash.
-    return getGravatarFromEmail(email, sizePhysicalPx);
```

Comment:
This commit can also delete this helper and similarly `getMediumAvatar`, which are effectively part of the old `getAvatarUrl` implementation. That way it's deleting the entire old implementation, which enhances the comparison this commit helps to make.